### PR TITLE
Remove OS X crap from repository

### PR DESCRIPTION
**Pull request by Zorgatone**
See https://github.com/tapio/rlutil/issues/14#issuecomment-158401555

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,9 +9,9 @@ test: rlutil.h test.cpp
...
 example-c: rlutil.h example.c
 	$(CC) $(CFLAGS) -o example-c example.c
-	
+
 .PHONY: clean
 
 clean:
 	rm -f test example-c
-
+	rm -rf *.dSYM
```
Comment (nabijaczleweli) [inline Makefile]: Are `*.dSYM` files directories? If not, why use the `-r` flag here?

Comment (Zorgatone) [inline Makefile]: They are directories

Comment (nabijaczleweli) [inline Makefile]: What kind of a world is this?

Curse you, Apple!

**Comment by nabijaczleweli** [comment]
:+1: :shipit:

**Comment by Zorgatone** [comment]
LOL:

```
TR: ~/repos/rlutil tommaso$ make
g++ -I. -g -O2 -Wall -Wextra -o test test.cpp
gcc -I. -g -O2 -Wall -Wextra -o example-c example.c
TR: ~/repos/rlutil tommaso$ ls
Makefile		example-c		rlutil.h
README.markdown		example-c.dSYM		test
cross-compile.sh	example.c		test.cpp
docs			index.html		test.dSYM
TR: ~/repos/rlutil tommaso$ ls test.dSYM
Contents
TR: ~/repos/rlutil tommaso$ ls test.dSYM/Contents
Info.plist	Resources
TR: ~/repos/rlutil tommaso$ ls test.dSYM/Contents/Resources
DWARF
TR: ~/repos/rlutil tommaso$ 
```

**Comment by nabijaczleweli** [comment]
y u do dis, Applel

**Comment by nabijaczleweli** [comment]
:shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit:

**Comment by Zorgatone** [comment]
[**@nabijaczleweli**](https://github.com/nabijaczleweli) commented on [20 nov 2015, 14:58 CET](https://github.com/tapio/rlutil/pull/26#issuecomment-158409221 "2015-11-20T13:58:27Z - Replied by Github Comment Enhancer"):
> :shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit:

APPEL

**Comment by nabijaczleweli** [comment]
Those :shipit:s are bigger than I expected.

meme's 2dank4me

**Comment by Zorgatone** [comment]
I just replied your comment...

They grew and populated: we're DOOMED

**Comment by nabijaczleweli** [comment]
QUICK, @tapio MERGE THIS AND SAVE US!

**Comment by Zorgatone** [comment]
@nabijaczleweli isn't it weird, there isn't any other way to communicate on github other than issues and comments?

Y NO CAHT GHITUB?

**Comment by Zorgatone** [comment]
I just realized @nabijaczleweli has now more commits on this repo than @tapio

**Comment by nabijaczleweli** [comment]
@Zorgatone There's no need for one

**Comment by Zorgatone** [comment]
[**@nabijaczleweli**](https://github.com/nabijaczleweli) commented on [20 nov 2015, 15:10 CET](https://github.com/tapio/rlutil/pull/26#issuecomment-158411575 "2015-11-20T14:10:51Z - Replied by Github Comment Enhancer"):
> [@Zorgatone](https://github.com/Zorgatone) There's no need for one

It feels unnatural and dirty to pollute issues with unrelated idiot comments :laughing:

**Comment by nabijaczleweli** [comment]
@Zorgatone I actually do , lol

**Comment by Zorgatone** [comment]
No you dont
No you dont

**Comment by nabijaczleweli** [comment]
No, I don't, that's weekly

**Comment by Zorgatone** [comment]
Those are just the pull requests. I was so dumb I thought that was total

**Comment by nabijaczleweli** [comment]
Where are @tapio​s when you need one?

**Pull request merged**